content/docs/command-reference/gc.md

```diff
@@ -20,14 +20,20 @@ DVC cache but are no longer needed. With `--cloud` it also removes data in
 To avoid accidentally deleting data, it raises an error and doesn't touch any
 files if no scope options are provided. It means it's user's responsibility to
 explicitly provide the right set of options to specify what data is still needed
-(so that DVC can figure out what fils can be safely deleted).
+(so that DVC can figure out what files can be safely deleted).
 
 One of the scope options (`--workspace`, `--all-branches`, `--all-tags`,
 `--all-commits`) or a combination of them must be provided. Each of them
 corresponds to keeping the data for the current workspace, and for a certain set
 of commits (determined by reading the DVC-files in them). See the
 [Options](#options) section for more details.
 
+> Please note that `dvc gc` pulls `.dir` >
```

Comment:
@shcheklein pre commit hook is adding ">" at end of this line.

Reply:
Hi. This is resolved right? Doesn't seem correct.

Reply:
I have added `.dir` to next line. The problem seems with using backtick " ` " at end of line(in a note). Pre commit hook doesn't except this.

Reply:
Strange. Never saw that before. Good to know...